api/src/api/notes.js

```diff
@@ -12,29 +13,59 @@ const {
   validateReqParams,
 } = require('../middleware/notes');
 
+const memberFromId = async (ids) => {
+  const memberPromises = ids.map((memberId) => {
```

Comment:
we can make this more concise by dropping the return statement/body of this function (e.g. `(memberId) => Member.findById(memberId)`)